ksqldb-engine/src/main/java/io/confluent/ksql/util/SandboxedSharedKafkaStreamsRuntimeImpl.java

```diff
@@ -42,7 +43,7 @@ public SandboxedSharedKafkaStreamsRuntimeImpl(
     );
 
     for (BinPackedPersistentQueryMetadataImpl query : sharedRuntime.collocatedQueries.values()) {
-      //kafkaStreams.addNamedTopology(query.getTopology());
```

Comment:
I think this might still be an issue for the validation runtimes because we want to build the topology with a new NamedTopologyBuilder that we got from the validation runtime, not the one based on the original query's actual runtime. But if the tests pass for now this is ok with me and we can clean it up later (it might technically work but is a bit awkward)

Reply:
yeah I have an idea how to clean it up, I think we might need to have the copy method take a parameter. but we should do that as a follow up

Reply:
sorry yes to clarify I'm not asking for any changes to this in the current PR